apps/frontend/src/modules/auth/slices/actions.ts

```diff
@@ -1,8 +1,12 @@
 import { createAsyncThunk } from "@reduxjs/toolkit";
 
+import { NotificationMessage } from "~/libs/enums/enums.js";
+import { notification } from "~/libs/modules/notification/notification.js";
```

Comment:
Should come from extraArguments
Change for storage as well

Reply:
Should we move StorageKey to frontend/src/libs/enums?

Reply:
It feels weird that I can use it inside of redux actions while it is in libs folder for storage module

Reply:
No, should be in ~/libs/modules/storage/storage.js